index.html

```diff
@@ -8,26 +8,39 @@
     <title>Personal Portfolio Website</title>
   </head>
   <body>
-     <!-- Main navigation bar or toolbar -->
-    <header class="main-nav">
-      <div class="brand-logo">
-        <a href="#">
-            <img class="logo-spin" src="assets/name-circle-r.svg" alt="Spinning Logo">
-            <img class="logo-still" src="assets/SMILE0.svg" alt="Still Logo">
-        </a>
-    </div>    
-      <nav>
-        <!-- Navigation menu with links to different sections of the page -->
-        <div class="nav-menu">
-          <button type="button" class="menu-icon"><img src="assets/icon-hamburger.svg" alt="Menu icon"></button>
-          <ul class="nav-links">
-            <li><a>Portfolio</a></li>
-            <li><a>About</a></li>
-            <li><a>Contact</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
+ <!-- Main navigation bar or toolbar -->
+<header class="main-nav">
+  <div class="brand-logo">
+    <a href="#">
+        <img class="logo-spin" src="assets/name-circle-r.svg" alt="Spinning Logo">
+        <img class="logo-still" src="assets/SMILE0.svg" alt="Still Logo">
+    </a>
+  </div>    
+  <nav>
+    <!-- Navigation menu with links to different sections of the page -->
+    <div class="nav-menu">
+      <button type="button" class="menu-icon" id="hamburger-icon"><img src="assets/icon-hamburger.svg" alt="Menu icon"></button>
+      <ul class="nav-links">
+        <li><a>Portfolio</a></li>
+        <li><a>About</a></li>
+        <li><a>Contact</a></li>
+      </ul>
```

Comment:
- [ ] As a best practice and in pursuit of accessibility for screen readers, and in fact the page's ease of navigation, I do not think that including a `href` in an anchor tag `a` is optional hence it is not different from `p`. Kindly add attributes `href` to these and ensure you have the corresponding `id`s to navigate the entire page section by section.